src/Parser/Warrior/Fury/Modules/Features/Checklist/Component.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import Checklist from 'Parser/Core/Modules/Features/Checklist2';
+import Rule from 'Parser/Core/Modules/Features/Checklist2/Rule';
+import Requirement from 'Parser/Core/Modules/Features/Checklist2/Requirement';
+import PreparationRule from 'Parser/Core/Modules/Features/Checklist2/PreparationRule';
+import GenericCastEfficiencyRequirement from 'Parser/Core/Modules/Features/Checklist2/GenericCastEfficiencyRequirement';
+
+class FuryWarriorChecklist extends React.PureComponent {
+  static propTypes = {
+    castEfficiency: PropTypes.object.isRequired,
+    combatant: PropTypes.shape({
+      hasTalent: PropTypes.func.isRequired,
+      hasTrinket: PropTypes.func.isRequired,
+    }).isRequired,
+    thresholds: PropTypes.object.isRequired,
+  };
+
+  render() {
+    const { combatant, castEfficiency, thresholds } = this.props;
+
+    const AbilityRequirement = props => (
+      <GenericCastEfficiencyRequirement
+        castEfficiency={castEfficiency.getCastEfficiencyForSpellId(props.spell)}
+        {...props}
+      />
+    );
+
+    return (
+      <Checklist>
+        <Rule
+          name="Use cooldowns effectively"
+          description={(
```

Comment:
Since you don't use any React elements, you can simplify it to `description="Your cooldowns ......"`